WordPress/Classes/ViewRelated/Blog/Site Picker/SitePickerViewController+QuickStart.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+private var alertWorkItem: DispatchWorkItem?
+private var observer: NSObjectProtocol?
+
+extension SitePickerViewController {
```

Comment:
I tested QuickStart with this new `SitePickerViewController` and the glowing signal is not appearing at all. This is how it is with the flag disabled:

<img width="384" alt="Screen Shot 2022-01-20 at 12 35 03" src="https://user-images.githubusercontent.com/7040243/150370645-650dd91b-0407-46a5-8bb1-e3cecab1280b.png">

I'm still able to complete the quick start steps, though.

Reply:
Fixed! 72d2296